packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -363,3 +425,95 @@ pub struct Plan {
     /// **info** is any application specific upgrade info to be included on-chain
     pub info: String,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+/// UpgradeProposal defines the struct for  upgrade proposal.
+pub struct UpgradeProposal {
+    /// **title** is a text title of proposal. Non unique.
```

Comment:
What does "non unique" mean? If you want to state that a value doesn't have to be unique, you should write "Doesn't have to be unique". Otherwise it sounds like it is never unique, which is probably not true.